themes-default/slim/package.json

```diff
@@ -111,29 +112,29 @@
       "$",
       "MEDUSA",
       "api",
-      "apiv1",
-      "apiRoute",
       "jQuery",
       "axios",
       "webRoot",
       "apiRoot",
-      "apiKey",
-      "Vuex"
+      "apiKey"
     ],
     "ignores": [
       "static/js/lib/**",
       "static/js/*.min.js",
       "static/js/vender.js",
       "build/**",
       "gulpfile.js",
-      "static/js/templates/name-pattern.vue"
+      "static/js/templates/**/*.vue"
```

Comment:
Didn't we already fix this?

Reply:
Maybe stuck in a PR?

Reply:
> Didn't we already fix this?

#4706 

> Maybe stuck in a PR?

What?
Please revert it to `"static/js/templates/name-pattern.vue"`

Reply:
Oow. Now i understand. Woops